### ISSUE TITLE
Sudden black screen

### ISSUE THREAD

**Issue by ilceltico** [nicoco007/WpfKenBurns#1]
I couldn't find a decent, simple, Ken Burns screensaver for Windows that didn't ask for 15 dollars or more. This tool is great, free, and much smoother than other free tools I tried!
Unfortunately the screen turns black after a couple of images, and restarting the slideshow or the PC doesn't help.
I downloaded the latest build on Oct 25th 2020 and Installed it on SysWOW64, running on Windows 10 Pro 64bit, version 1909, build 18363.1139. I have two monitors, which appeared to work properly for the first 2 transitions, I don't know if this helps.

Feel free to ask any other information you need

**Comment by nicoco007** [comment]
Oh, thanks! I honestly wasn't really expecting anyone to use this so it's definitely not as polished as it could be. I do have a few questions:

- Could you attach a screenshot of your configuration window?
- Does it stop working after a specific number of images, after a specific amount of time, or is it completely random?

**Comment by ilceltico** [comment]
Thank you for the reply!
Do you mean the configuration of the Ken Burns or the PC? For the Ken Burns, I left it at default values. 
It worked for 2 transitions at most, but most of the times it worked for 1 or even none, it just started black and stayed black.

After more research, I've found this old website http://alumni.cs.ucr.edu/~gstitt/motionpicture/
I get some stutters from time to time, but it still works on Windows 10!
They said they also had to fix a crash for multiple monitors, so I guess that my 2-monitor setup is very likely to be the cause

**Comment by nicoco007** [comment]
Yes sorry, I meant the configuration of WpfKenBurns. I'm also running 2 monitors (and previously tested it on 3) so I don't think that's the issue.

Are you still interested in fixing your issue or are you going with the other screensaver you found?

**Comment by ilceltico** [comment]
I'm going with the other one right now so there's no need to fix anything, but I found your work to be cool and I gave it a star :D. Maybe I was not the only one looking for a decent screensaver for a new monitor. Your choice, no pressure at all!

**Comment by jplusc** [comment]
I know this is old ... but I will mention this anyway incase it helps anyone else.

I also had Sudden black screen's with this program, so I just added a MessageBox where the exception is caught and application shutdown in WindowSynchronizer.cs and I found some images/files cause a:

`NotSupportedException - no imaging component suitable to complete this operation` exception.

you have to go thru your directories and weed these out.

a stray txt or html file will crash it everytime.

I didn't play with it enough to see which function is causing the exception (probably GetImage())  but the exception can probably be caught there so the file can simply be skipped.


In any case .. thanks for making and sharing this.. it came in handy for me at least!

**Comment by nicoco007** [comment]
Well that's a pretty stupid oversight on my part. I'll make sure to get that fixed.

**Comment by nicoco007** [comment]
Should be fixed by f3dfc1b11a1ee76f81cc0164bf82a60045f797e4
